slang.h

```diff
@@ -313,22 +313,40 @@ extern "C"
     spGetTranslationUnitCount(
         SlangCompileRequest*    request);
 
-    /** Get the output code associated with a specific translation unit
+    /** Get the output code associated with a specific translation unit.
 
     The lifetime of the output pointer is the same as `request`.
     */
     SLANG_API char const* spGetTranslationUnitSource(
         SlangCompileRequest*    request,
         int                     translationUnitIndex);
 
-    /** Get the output code associated with a specific entry point.
+    /** Get the output bytecode associated with a specific translation unit.
+
+    The lifetime of the output pointer is the same as `request`.
+    */
+    SLANG_API void const* spGetTranslationUnitCode(
```

Comment:
We probably don't need `spGetTranslationUnitCode`. In fact, I'm probably going to deprecate `spGetTranslationUnitSource` at some point